packages/grafana-data/src/panel/registryFactories.ts

```diff
@@ -0,0 +1,58 @@
+import { FieldConfigOptionsRegistry } from '../field/FieldConfigOptionsRegistry';
+import { standardFieldConfigEditorRegistry } from '../field/standardFieldConfigEditorRegistry';
+import { FieldConfigProperty } from '../types/fieldOverrides';
+import { FieldConfigEditorBuilder } from '../utils/OptionsUIBuilders';
+import { SetFieldConfigOptionsArgs } from './PanelPlugin';
+
+export function createFieldConfigRegistry<TFieldConfigOptions>(
```

Comment:
Shold this have jsdoc state tag? something like 

/** @beta */

Reply:
Yupp, just wanted to get your input first.